test/Backends/ArithmeticCircuit.hs

```diff
@@ -28,10 +29,34 @@ equalTerm = J.Lam "x" (J.Var "x")
 equalType :: Type
 equalType = J.Pi Omega (J.PrimTy ()) (J.PrimTy ())
 
-equalCircuit :: C.ArithCircuit Fr
-equalCircuit =
-  C.ArithCircuit
-    { C.weights = C.GateWeights {C.wL = [], C.wR = [], C.wO = []},
-      C.commitmentWeights = [],
-      C.cs = []
-    }
+power :: Term
```

Comment:
This is proving knowledge of a square-root, right?

Reply:
It is, sorry!! My arithmetic is broken, I just guide myself by algebraic laws and hope for the best.